en/docs/develop-components/manage-component-source-configurations.md

```diff
@@ -114,6 +129,23 @@ Click the respective tab to view the structure for your current configuration fi
 
         To use custom environment variable names instead of Choreo's default ones, add the dependency as a `serviceReference` in your `component.yaml v1.1` file. You can copy the `serviceReference` section from the `component.yaml v1.0` tab and paste it under `dependencies` in your `component.yaml v1.1` file, which maintains backward compatibility with the v1.0 format.
 
+    ### Runtime configurations
+    In the `configuration` section of the `component.yaml` file, you can define runtime configurations for the component. Currently, the configurations support env variable injection related to dependencies.
+    
+    | Configuration                 | Required     | Description                                                                      |
+    |-------------------------------|--------------|----------------------------------------------------------------------------------|
+    | **env**                       | Optional     | Array of env variable confiuration                                               |
+    | **&nbsp;name**                | Required     | A unique name starting with a letter or an underscore, containing letters, numbers and underscore for the environment variable name |
+    | **&nbsp;valueFrom**           | Required     | Value source for the env variable                                                |
+    | **&nbsp;&nbsp;connectionRef** | Required     | Connection reference value source definition                                     |
+    | **&nbsp;&nbsp;&nbsp;name**    | Required     | Choreo connection name to refer the value from                                   |
+    | **&nbsp;&nbsp;&nbsp;key**     | Required     | Choreo connection configuration key to refer the value from. Refer to [connection configurations](https://wso2.com/choreo/docs/develop-components/sharing-and-reusing/use-a-connection-in-your-service/) or see in-line in-line developer guide displayed during connection creation to discover available keys                   |
```

Comment:
Suggestion:
"The Choreo connection configuration key to reference the value from. For details on obtaining available keys, see [Consume a service through a connection](https://wso2.com/choreo/docs/develop-components/sharing-and-reusing/use-a-connection-in-your-service/#consume-a-service-through-a-connection) or the in-line developer guide."